src/main/java/dk/kb/util/yaml/YAML.java

```diff
@@ -570,27 +617,30 @@ public String getString(String path, String defaultValue) throws NullPointerExce
         }
     }
 
-    /**
-     * Resolves all values related to a given key, from a YAML structure. All values that have the specified key are
-     * returned as part of a list.
-     * @deprecated
-     * This is no longer the best method to visit values. Use {@link #visit(Object, YAML, YAMLVisitor)} instead.
-     * @param key the key to look for in the input YAML.
-     * @return a list of all values that can be cast to strings.
-     */
-    @Deprecated
-    public List<Object> getMultiple(String key){
-        String correctedPath = "**." + key;
-        MultipleValuesVisitor visitor = new MultipleValuesVisitor();
-        visit(correctedPath, this, visitor);
-        return visitor.extractedValues;
-    }
+    // TODO: Evaluate the redefinition of getMultiple
+    // The old getMultiple 
+
+//    /**
+//     * Resolves all values related to a given key, from a YAML structure. All values that have the specified key are
+//     * returned as part of a list.
+//     * @deprecated
+//     * This is no longer the best method to visit values. Use {@link #visit(Object, YAML, YAMLVisitor)} instead.
+//     * @param key the key to look for in the input YAML.
+//     * @return a list of all values that can be cast to strings.
+//     */
+//    @Deprecated
+//    public List<Object> getMultiple(String key){
+//        String correctedPath = "**." + key;
+//        MultipleValuesVisitor visitor = new MultipleValuesVisitor();
+//        visit(correctedPath, this, visitor);
+//        return visitor.extractedValues;
+//    }
```

Comment:
Please remove the old implementation completely. I agree with you and @jorntx on the change.